tools/mtouch/Makefile

```diff
@@ -158,6 +158,8 @@ SIMLAUNCHER_FRAMEWORKS =  \
 	-weak_framework QuickLookThumbnailing		\
 	-weak_framework SoundAnalysis			\
 	-weak_framework VisionKit			\
+						\
```

Comment:
is the extra line here needed?

Reply:
We separate frameworks added by different Xcode releases so not really needed but follows the pattern of the file